Tests/js/views/fields/assets/ez-image-editview-tests.js

```diff
@@ -464,6 +464,115 @@ YUI.add('ez-image-editview-tests', function (Y) {
+                    });
+            
+                eventFacade.target = new Mock();
+                Mock.expect(eventFacade.target, {
+                    method: 'getDOMNode',
+                    returns: {files: [file]},
+                });
+                Mock.expect(eventFacade.target, {
+                    method: 'set',
+                    args: ['value', ''],
+                });
+                Mock.expect(fileReader, {
+                    method: 'readAsDataURL',
+                    args: [file],
+                    run: function (f) {
+                        fileReader.result = base64ImgContent;
+                        fileReader.onload({target: fileReader});
+                    }
+                });
+            
+                this.view.render();
+                this.view._updateFile(eventFacade);
+                Assert.isFalse(
+                    this.view.get('warning'),
+                    "The warning attribute should stay false"
+                );
+            
+            
+                this.view.after('fileChange', this.next(Y.bind(function (e) {
+                        binaryFile = this.view.get('file');
+            
+                        Assert.areEqual(
+                            fileContent, binaryFile.data,
+                            "The binaryfile content should be available in the file attribute"
+                        );
+                        Assert.areEqual(
+                            file.name, binaryFile.name,
+                            "The binaryfile name should be available in the file attribute"
+                        );
+                        Assert.areEqual(
+                            file.size, binaryFile.size,
+                            "The binaryfile size should be available in the file attribute"
+                        );
+                        Assert.areEqual(
+                            file.type, binaryFile.type,
+                            "The binaryfile type should be available in the file attribute"
+                        );
+                        Assert.isUndefined(
+                            fileReader.onload,
+                            "The onload handler should be resetted"
+                        );
+                },this)));
+                this.wait();
+            },
+
+            "Should refuse an image than can NOT be loaded": function () {
+                var win = Y.config.win,
+                    fileReader = this.view.get('fileReader'),
+                    file = {size: 5 * this.multiplicator, name: "test-image.gif", type: "image/gif"},
+
+                    eventFacade = new Y.DOMEventFacade({
+                        type: 'change'
+                    });
+
+                this.originalURL = win.URL;
+                win.URL = {};
+                win.URL.createObjectURL = function (uri) {};
+
+                eventFacade.target = new Mock();
+                Mock.expect(eventFacade.target, {
```

Comment:
I'm not overriding binaryfile test cases anymore so now it's the only test using this mocks. That's why I believe I can now keep them where they are.